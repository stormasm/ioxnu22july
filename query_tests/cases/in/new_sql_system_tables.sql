-- IOX_SETUP: TwoMeasurementsManyFieldsTwoChunks

-- validate we have access to information schema for listing system tables
-- IOX_COMPARE: sorted
SELECT * from information_schema.tables where table_schema = 'system';

-- IOX_COMPARE: sorted
SELECT issue_time, query_type, query_text, success FROM system.queries;
