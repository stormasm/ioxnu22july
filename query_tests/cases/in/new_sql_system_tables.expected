-- Test Setup: TwoMeasurementsManyFieldsTwoChunks
-- SQL: SELECT * from information_schema.tables where table_schema = 'system';
-- Results After Sorting
+---------------+--------------+------------+------------+
| table_catalog | table_schema | table_name | table_type |
+---------------+--------------+------------+------------+
| public        | system       | queries    | BASE TABLE |
+---------------+--------------+------------+------------+
-- SQL: SELECT issue_time, query_type, query_text, success FROM system.queries;
-- Results After Sorting
+----------------------+------------+------------+---------+
| issue_time           | query_type | query_text | success |
+----------------------+------------+------------+---------+
| 1970-01-01T00:00:00Z | sql        | SELECT 1;  | true    |
+----------------------+------------+------------+---------+
